posthog/test/test_rate_limit.py

```diff
@@ -186,6 +232,19 @@ def test_rate_limits_work_on_non_team_endpoints(self, rate_limit_enabled_mock, i
             },
         )
 
+    @patch("posthog.rate_limit.PassThroughBurstRateThrottle.rate", new="5/minute")
+    @patch("posthog.rate_limit.statsd.incr")
+    @patch("posthog.rate_limit.is_rate_limit_enabled", return_value=True)
+    def test_does_not_rate_limit_non_personal_api_key_endpoints(self, rate_limit_enabled_mock, incr_mock):
```

Comment:
It'd be nice if this test tested being rate limited on the personal api key but still being able to make calls using auth

Reply:
ah, makes sense